pkg/component/etcd/poddisruptionbudget/valueshelper.go

```diff
@@ -33,8 +34,8 @@ func GenerateValues(etcd *druidv1alpha1.Etcd) Values {
 	}
 
 	annotations := map[string]string{
-		ownedByAnnotationKey:   fmt.Sprintf("%s/%s", etcd.Namespace, etcd.Name),
-		ownerTypeAnnotationKey: "etcd",
+		common.GardenerOwnedBy:   fmt.Sprintf("%s/%s", etcd.Namespace, etcd.Name),
```

Comment:
These changes are completely unrelated to etcdcopybackuptask and should not be clubbed with this PR. Please remove this and file a separate PR for this.